app/src/App.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import Board from './Board'
+import './App.css'
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      game: new Board(25, 25),
+      playInterval: null,
+    }
+    this.click = this.click.bind(this)
+    this.step = this.step.bind(this)
+    this.play = this.play.bind(this)
+    this.randomize = this.randomize.bind(this)
+    this.clear = this.clear.bind(this)
+  }
+
+  click(event) {
+    const aliveState = event.target.className
+    const coord = event.target.getAttribute('dataset').split(',')
+    const row = +coord[0]
+    const col = +coord[1]
+    if (aliveState === 'alive') {
+      //tbd
+    }
+  }
+
+  step() {
+    if (this.state.playInterval) clearInterval(this.state.playInterval)
+    const newGame = Object.assign(this.state.game)
+    newGame.tick()
+    this.setState({ game: newGame })
+  }
+
+  play() {
+    this.setState({
+      playInterval: setInterval(() => {
+        this.step()
+      }, 500),
+    })
+  }
+
+  randomize() {
+    if (this.state.playInterval) clearInterval(this.state.playInterval)
+    for (let i = 0; i < this.state.game.rows; i++) {
+      for (let j = 0; j < this.state.game.cols; j++) {
+        const newGame = Object.assign(this.state.game)
```

Comment:
Better to use a spread operator instead of `Object.assign()`: https://thecodebarbarian.com/object-assign-vs-object-spread.html

Reply:
I tried doing that but it doesn't copy over the Board class methods I want, but Object.assign does.